### ISSUE TITLE
the loss is negative 

### ISSUE THREAD

**Issue by zzdang** [salu133445/musegan#7]
---exps/temporal_hybrid_GPU_1--- epoch:  0 | batch:    1/ 785 | time:  36.23 
D loss: 15880.42, G loss: -241.65

I train your dataset use the main.py code,but the loss ins negative, anything is wrong?

**Comment by lilacheson** [comment]
Hey guy, do you know how to get the training data in .npy format?

**Comment by wayne391** [comment]
Hello, I found there are some bugs in the Github version codes. We've fixed them.  Pleace check out the new version.

**Comment by salu133445** [comment]
Fixed by 9a397b0 and efc4b02.
